website/src/contribute/release-guide.md

```diff
@@ -1248,6 +1256,14 @@ please follow [the guide](https://help.github.com/articles/creating-a-personal-a
    delete the `.asc`, `.sha512`;
 3. Upload the new release `twine upload *` from the directory with the `.zip` and `.whl` files;
 
+[Installing twine](https://packaging.python.org/tutorials/packaging-projects/):
```

Comment:
Was this the intended URL?

Reply:
Yes. Will make it more detailed.

Reply:
Okay - I thought you meant to link to the twine homepage.

Reply:
This page describes how packaging works for python, including using twine for publishing binaries. I find it more useful.